action/bucket.go

```diff
@@ -15,7 +14,11 @@ func MakeBucket(remote string) (err error) {
 		return
 	}
 	// Init bucket
-	if err = helper.PutBucket(bucketName, contexts.Zone); err != nil {
+	err = contexts.Storage.SetupBucket(bucketName, contexts.Zone)
```

Comment:
You cannot `SetupBucket` for a non-exists bucket which I thought would always return "Detect bucket location failed". The command is for making a new bucket.